src/records/Project.js

```diff
@@ -15,6 +15,7 @@ export default class Project extends Record({
   hiddenUIComponents: new Set(['console']),
   updatedAt: null,
   instructions: '',
+  repoName: null,
```

Comment:
Maybe premature generalization, but this would feel better to me inside a collection or record of some sort whose role is to store “external locations that this project has been exported to”. Might also dovetail with @pwjablonski’s work on Google Classroom integration, since that’s another case where a project needs to remember a link to an external resource?